docs/book/src/tasks/migrating-to-user-managed-cert-manager.md

```diff
@@ -0,0 +1,16 @@
+# Migrating to user-managed cert-manager
+
+You may want to migrate to a user-managed cert-manager further down the line, after initialising cert-manager on the management cluster through `clusterctl`.
+
+`clusterctl` looks for the label `clusterctl.cluster.x-k8s.io/core=cert-manager` on all api resources in a namespace. If it finds the label, `clusterctl` will manage the cert-manager deployment. If you want to manage and install your own cert-manager, you'll need to remove this label from all API resources. You can list all the resources with that label by running:
+```bash
+kubectl get all -A --selector=clusterctl.cluster.x-k8s.io/core=cert-manager
+```
+
```

Comment:
Would moving or repeating this line, or something like it, after the above command be clearer?

`If you want to manage and install your own cert-manager, you'll need to remove this label from all API resources.`